pyproject.toml

```diff
@@ -5,10 +5,39 @@
 # >>> python -m build
 
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.0"]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.0", "poetry-core>=1.0.0"]
 build-backend = "setuptools.build_meta"
 
 # Get version number from git tag
 # See https://github.com/pypa/setuptools_scm/
 [tool.setuptools_scm]
 write_to = "resqpy/version.py"
+
+[tool.poetry]
+name = "resqpy"
+version = "1.6.1"
+description = "Python API for working with RESQML models"
+authors = ["BP"]
+license = "MIT"
+
+[tool.poetry.dependencies]
```

Comment:
These dependencies seem far too restrictive. Specification of exact versions is handled in the Poetry.lock file, and I do not think we should reproduce that here.

I believe this section should contain version _constraints_ only, not restricting all users to only use these exact versions. I would suggest using the Poetry style of "caret requirements":
https://python-poetry.org/docs/dependency-specification/

Reply:
I think same comment for dev dependencies below?